docs/02-Scales.md

```diff
@@ -310,14 +310,15 @@ The following additional configuration options are provided by the radial linear
 
 Name | Type | Default | Description
 --- | --- | --- | ---
-lineArc | Boolean | false | If true, circular arcs are used else straight lines are used. The former is used by the polar area chart and the latter by the radar chart
+labelPoints | Boolean | true | If true, angle lines and point labels are drawn.
```

Comment:
Yeah, it's sort of like those. I'll update the code to use those two display settings instead